src/components/ui/calendar.tsx

```diff
@@ -212,8 +221,9 @@ function CalendarDropdown(props: DropdownProps) {
     );
   }
 
-  return null;
+  return <div>dropdown</div>;
```

Comment:
Currently, dropdown is only ever used for the selection of `months` and `year`. Ofc, we'll have to revisit this later should there be a need to use it for more than the aforementioned options.